### PR TITLE
Design updates

### PR DESCRIPTION
**Pull request by cedricwaxwing**
Various Updates to standardize the UX and improve visual consistency.

### CODE REVIEW

**Comment by cedricwaxwing** [comment]
@juliopavila Do you think you could help me with something here?

I'd like to add the ability to remove a permission from the permission view (in addition to the permission modal). Do you think you could add that functionality to the "x" buttons I've added here?

image

**Comment by juliopavila** [comment]
> @juliopavila Do you think you could help me with something here?
> 
> I'd like to add the ability to remove a permission from the permission view (in addition to the permission modal). Do you think you could add that functionality to the "x" buttons I've added here?
> 
> image

Yes sure!

File: packages/app/src/components/views/publication/PermissionView.tsx [hunk 1]
```diff
@@ -276,6 +280,26 @@ export const PermissionView: React.FC = () => {
...
+                        </>
+                      )}
+                    />
+                    {errors && errors.account && (
+                      <FormHelperText sx={{ color: palette.secondary[1000], textTransform: "capitalize" }}>
+                        {errors.account.message}
+                      </FormHelperText>
+                    )}
+                  </Grid>
+                ))} */}
```
Comment (juliopavila) [inline packages/app/src/components/views/publication/PermissionView.tsx:302]: remove old code

File: packages/app/src/components/views/publication/PublicationView.tsx [hunk 2]
```diff
@@ -70,11 +61,19 @@ export const PublicationView: React.FC<PublicationViewProps> = ({ updateChainId
...
             <Grid item>
-              <PublicationContainer container gap={3} alignItems={"center"}>
-                <Grid item>
+              <Stack
+                gap={3}
+                direction={["column", "row"]}
+                sx={{
+                  alignItems: ["flex-start", "flex-start", "center"],
+                  justifyContent: ["center", "center", "flex-start"],
+                  // flexDirection: ["column", "row"],
```
Comment (juliopavila) [inline packages/app/src/components/views/publication/PublicationView.tsx:73]: remove old code

File: packages/app/src/components/views/publication/PublicationView.tsx [hunk 3]
```diff
@@ -83,25 +82,53 @@ export const PublicationView: React.FC<PublicationViewProps> = ({ updateChainId
...
-                </Grid>
-              </PublicationContainer>
+                      {/* <CopyAllOutlined
+                        className="copyIcon"
+                        sx={{
+                          color: palette.grays[600],
+                          width: 16,
+                          height: 16,
+                        }}
+                      /> */}
```
Comment (juliopavila) [inline packages/app/src/components/views/publication/PublicationView.tsx:122]: remove old code

**Review by juliopavila — CHANGES_REQUESTED** [review]
Great job, remove the old code and fix the build problem!

**Review by juliopavila — APPROVED** [review]

**Pull request merged**